patches/minecraft/net/minecraft/entity/EntityType.java.patch

```diff
@@ -20,7 +20,90 @@
     public static final EntityType<EntityAreaEffectCloud> field_200788_b = func_200712_a("area_effect_cloud", EntityType.Builder.func_201757_a(EntityAreaEffectCloud.class, EntityAreaEffectCloud::new));
     public static final EntityType<EntityArmorStand> field_200789_c = func_200712_a("armor_stand", EntityType.Builder.func_201757_a(EntityArmorStand.class, EntityArmorStand::new));
     public static final EntityType<EntityTippedArrow> field_200790_d = func_200712_a("arrow", EntityType.Builder.func_201757_a(EntityTippedArrow.class, EntityTippedArrow::new));
-@@ -411,7 +412,7 @@
+@@ -227,6 +228,11 @@
+    private String field_210762_aX;
+    @Nullable
+    private final Type<?> field_206832_aX;
++   private boolean useVanillaSpawning;
```

Comment:
Consider adding `/* Forge Start */` blocks, etc. so people won't confuse them for vanilla fields.